### PR TITLE
Hst updates

### PR DESCRIPTION
**Pull request by almannin**
updating json files with HST derived values, updating names, and lots of other changes

### CODE REVIEW

File: frb/data/Galaxies/20190608B/FRB20190608B_host.json [hunk 1]
```diff
@@ -138,17 +157,11 @@
...
-    },
-    "ra": 334.0204291666666,
-    "ra_FRB": 334.01987499999996,
+    "ra": 334.0204166666666,
+    "ra_FRB": 334.019875,
     "redshift": {
-        "z": 0.11778050000000001,
-        "z_FRB": 0.11778050000000001,
-        "z_spec": 0.11778050000000001
+        "z": 0.1177805,
```
Comment (ntejos) [inline frb/data/Galaxies/20190608B/FRB20190608B_host.json:163]: this is correct, Macquart+2020 reports 0.1178, but Mannings+ reports 0.1177.

**Review by ntejos — APPROVED** [review]
Left some easy to address comments; otherwise looks good

File: frb/associate/frbs.py [hunk 2]
```diff
@@ -77,13 +77,13 @@
     cand_separation=20*units.arcsec,
     plate_scale = 0.1616 * units.arcsec,
 )
-frb20180916 = {**base_config, **updates}  # Use | in 3.9
+FRB20180916 = {**base_config, **updates}  # Use | in 3.9
```
[the remark this answers is not in the document]
Comment (profxj) [reply frb/associate/frbs.py:80]: We have been using "A", is that wrong?

Have made this A for now

[the remark this answers is not in the document]
Comment (profxj) [reply frb/associate/frbs.py:80]: Nope, you are right.  Should be B and I've updated it accordingly

File: frb/associate/frbs.py [hunk 3]
```diff
@@ -137,12 +137,12 @@
     plate_scale = 0.28 * units.arcsec,
     cand_separation=10*units.arcsec,
 )
-frb20190523 = {**base_config, **updates}  # Use | in 3.9
+FRB20190523 = {**base_config, **updates}  # Use | in 3.9
```
[the remark this answers is not in the document]
Comment (profxj) [reply frb/associate/frbs.py:140]: corrected

File: frb/associate/frbs.py [hunk 4]
```diff
@@ -296,7 +296,7 @@
     cand_separation=10*units.arcsec,
     plate_scale = 0.1616 * units.arcsec,
 )
-frb20180301 = {**base_config, **updates}  # Use | in 3.9
+FRB20180301 = {**base_config, **updates}  # Use | in 3.9
```
[the remark this answers is not in the document]
Comment (profxj) [reply frb/associate/frbs.py:299]: Agreed, A.  Using that

**Comment by profxj** [comment]
merging with some apprehension.  am expecting 
there will be some more dust to settle..

**Pull request merged**